about/faq.md

```diff
@@ -0,0 +1,145 @@
+platforms. GstWebRTC depends on OpenSSL, which is released under an Apache-style
+license, so it doesn't have limitations regarding redistribution in binary form.
+
+## What's up with EME? How can I support this feature in my WPE-based product?
+
+There is code in WebKit to support EME but in any case you will need a license
+agreement with DRM CDM providers to access it since this part is not open source. There are three ways you can
+get this working:
+
+- Getting a license and use the [Thunder OCDM plugin](https://github.com/rdkcentral/Thunder).
+- Or write a Thunder compatible API complement that will glue with your DRM system.
+- Or write a new CDM backend for WebKit using your DRM system.
+
+## What is (and isn't) cog?
+
+Quoting [Cog's README](https://github.com/igalia/cog):
+
+> Cog is a small single “window” launcher for the WebKit WPE port. It is small,
+> provides no user interface, and is suitable to be used as a Web application
+> container. The “window” may be fullscreen depending on the WPE backend being
+> used.
+
+Cog's usage scenarios span from a MiniBrowser application to a full web-app
+container application meant to run HTML-based User-interfaces on embedded
+platforms and products.
+
+Although it can run on Linux-based desktop environments, Cog is not a full-blown
+Web Browser to be compared with Google Chrome or Safari. Cog's primary
+environment is on embedded platforms, it can run within a Wayland compositor
+such as Weston. Additionally if the platform supports KMS/DRM, Cog can run as a
+full-screen standalone browser, this use-case is very common on Kiosk products
+for instance.
+
+If you are a developer aiming to enable WPE on a certain embedded platform, Cog
+combined with WPEBackend-FDO provides the most flexible solution for agile
+tinkering and to test WPE's features.
+
+## Is Wayland required to run WPE?
+
+As we say in Galicia, "it depends". Read on for more details.
+
+[WPE's architecture](/about/architecture.html) was designed in order to decouple
+rendering out of the Web engine and delegate this task to rendering backends and
+to the application running the Web engine. So if your application requires a
+Wayland compositor, such as Weston, then you need to make sure you use a
+Wayland-EGL compatible rendering backend, such as WPEBackend-FDO for instance.
+Using this backend's API in your application will allow for smooth rendering in
+any Wayland compositor. [Cog](https://github.com/igalia/Cog) is one such
+WPE-based application relying on WPEBackend-FDO. If you don't need a Wayland
+compositor and just want to render a web-page full-screen, like in most Kiosk
+and STB use-cases, then you can render to KMS/DRM (Cog also supports this
+use-case).
+
+## Are open dialogs/popups menus supported?
+
+JavaScript popups and dialogs rendering is the duty of the WPE application. The
+reference WPE Browser, Cog, has limited support for these features (as of 2020,
+cog supports option menus).
```

Comment:
I am having some trouble to parse the first phrase here. Also plain HTML like `<input type="file">` can result in a file chooser dialog without involving any JavaScript so I think it's better to remove the name of the scripting language from here :wink:  

For example, how about this:

```suggestion
The application embedding WPE is responsible for rendering popups and dialogs.
The reference WPE Browser, Cog, has limited support for these features (as of 2021,
it supports option menus).
```